jekyll/_cci2/workflows.md

```diff
@@ -16,20 +16,22 @@ To increase the speed of your software development through faster feedback, shor
 
 ## Overview
 
-A **workflow** is a set of rules for defining a collection of jobs and their run order. Workflows support complex job orchestration using a simple set of configuration keys to help you resolve failures sooner.
+A **Workflow** is a set of rules for defining a collection of jobs and their run order. Workflows support complex job orchestration using a simple set of configuration keys to help you resolve failures sooner.
 
-With workflows, you can:
+With Workflows, you can:
 
 - Run and troubleshoot jobs independently with real-time status feedback.
-- Schedule workflows for jobs that should only run periodically.
+- Schedule Workflows for jobs that should only run periodically.
 - Fan-out to run multiple jobs in parallel for efficient version testing.
 - Fan-in to quickly deploy to multiple platforms.
 
-For example, if only one job in a workflow fails, you will know it is failing in real-time. Instead of wasting time waiting for the entire build to fail and rerunning the entire job set, you can rerun *just the failed job*.
+For example, if only one job in a Workflow fails, you will know it is failing in real-time. Instead of wasting time waiting for the entire build to fail and rerunning the entire job set, you can rerun *just the failed job*.
+
+Projects configured with Workflows often include multiple jobs that share similar or identical syntax (Docker images, environment variables, particular `run` steps); with [YAML Anchors/Aliases](http://yaml.org/spec/1.2/spec.html#id2765878), you can [reference these shared YAML snippets repeatedly throughout your `.circleci/config.yml` file](https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/), in order to keep your `config.yml` small and DRY.
```

Comment:
Personal opinion: parentheses pull readers out of the context of a sentence. If these are _actually_ important, I recommend giving them their own sentence.

For links in docs, our standard is to be [explicit](https://github.com/circleci/circleci-docs/blob/master/CONTRIBUTING.md#explicit-names) about what you're linking _to_.